linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -101,8 +101,8 @@ def _install_kmp(self, inputfile, online, language, area):
         self.kmpdocdir = get_keyman_doc_dir(area, self.packageID)
         self.kmpfontdir = get_keyman_font_dir(area, self.packageID)
 
-        if not os.path.isdir(self.packageDir):
-            os.makedirs(self.packageDir)
+        if not self._safeMakeDirs(self.packageDir):
```

Comment:
This used to check a boolean. Is the same as 
`if self._safeMakeDirs(self.packageDir) is None`?